tests/apollo/util/bft.py

```diff
@@ -501,11 +501,15 @@ def start_replica(self, replica_id):
             if self.is_existing and self.config.stop_replica_cmd is not None:
                 self.procs[replica_id] = self._start_external_replica(replica_id)
             else:
+                my_env = os.environ.copy()
+                # Uncomment the next line to run Apollo tests with ThreadSanitizer
+                # my_env["TSAN_OPTIONS"] = "history_size=7 external_symbolizer_path=/usr/lib/llvm-6.0/bin/llvm-symbolizer"
```

Comment:
Might be we can put this variable in apollo/CMakeLists.txt in the APOLLO_TEST_ENV?

Reply:
As discussed, we moved this into APOLLO_TEST_ENV